extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -19,15 +19,21 @@ import java.util.*
  * `["format",["coalesce",["get","name_en"],["get","name"]],{}]` will be replaced by
  * `["format",["coalesce",["get","name_zh-Hans"],["get","name"]],{}]`
  */
-internal fun setMapLanguage(locale: Locale, style: StyleInterface) {
+internal fun setMapLanguage(locale: Locale, style: StyleInterface, layerIds: List<String>?) {
   style.styleSources
     .filter { sourceIsFromMapbox(style, it) }
     .forEach { source ->
       style.styleLayers
         .filter { it.type == LAYER_TYPE_SYMBOL }
+        .filter { layer ->
+          layerIds?.contains(layer.id) ?: true
+        }
         .forEach { layer ->
           val symbolLayer = style.getLayerAs<SymbolLayer>(layer.id)
           symbolLayer.textFieldAsExpression?.let { textFieldExpression ->
+            if (BuildConfig.DEBUG) {
```

Comment:
I don't remember using that anywhere else in project. Do we really need that logging? I'd remove it, we can always add it manually if we really need it.

Reply:
Actually, I have added and removed the log output many times and these logs are really important for developing. Is there any particular reason we can't use this?

Reply:
I don't have any particular reasons.
@mapbox/maps-android WDYT?

Reply:
If this message isn't printed frequently, I'd keep the log. And probably we need a centralised solution to filter out log messages for specific plugins.

Reply:
There would be about a dozen of log output as we have several symbol layers in the style.

Reply:
I'm fine with using BuildConfig.DEBUG, users won't ever hit this. The filter log feature is being developed upstream atm.